src/qt/forms/receivecoinsdialog.ui

```diff
@@ -174,6 +161,73 @@
           </property>
          </widget>
         </item>
+        <item row="5" column="2">
+         <layout class="QHBoxLayout" name="horizontalLayout_6">
+          <item>
+           <widget class="BitcoinAmountField" name="reqAmount">
+            <property name="sizePolicy">
+             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+              <horstretch>0</horstretch>
+              <verstretch>0</verstretch>
+             </sizepolicy>
+            </property>
+            <property name="minimumSize">
+             <size>
+              <width>80</width>
+              <height>0</height>
+             </size>
+            </property>
+            <property name="maximumSize">
+             <size>
+              <width>1000</width>
+              <height>100</height>
+             </size>
+            </property>
+            <property name="toolTip">
+             <string>An optional amount to request. Leave this empty or zero to not request a specific amount.</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QCheckBox" name="useBech32">
+            <property name="sizePolicy">
+             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+              <horstretch>0</horstretch>
+              <verstretch>0</verstretch>
+             </sizepolicy>
+            </property>
+            <property name="maximumSize">
+             <size>
+              <width>1000</width>
+              <height>16777215</height>
+             </size>
+            </property>
+            <property name="focusPolicy">
+             <enum>Qt::StrongFocus</enum>
+            </property>
+            <property name="toolTip">
+             <string>Bech32 addresses (BIP-173) are cheaper to spend from and offer better protection against typos. Not all wallets support this new format and your privacy may be impacted if not enough other people use it.</string>
```

Comment:
Can this also say that not checking this will use the default address type and what that address type is?

Reply:
@achow101  it's going to become a bit of a mouthful. I propose taking out the language about privacy(not really actionable information imo), and more information about functional behavior as you said.

Reply:
@achow101 my idea was to show change the tooltip depending on what the wallet default is: https://github.com/bitcoin/bitcoin/pull/11991#issuecomment-356950319 That should keep it short enough.

> Can this also say that not checking this will use the default address type (default or from bitcoin.conf/command line options) and what that address type is

That's not what it does. It's bech32 when on, p2sh-segwit when off. When launched with legacy address type it's disabled and unchecked, so hopefully clear enough that it won't use p2sh-segwit in that case.

Reply:
@instagibbs I can leave out the privacy remark, but I do think it's actionable: people can google "bech32 privacy" (and find nothing for the moment...). I think it's useful to point out that being the first to use a new address format makes you stand out.

Reply:
Probably best in release notes, imo. This information could quickly become false, depending on uptake.

Reply:
@Sjors I don't think it is clear that it will be using p2sh-segwit when unchecked. It's trivial to just add a note to the tooltip indicating as such.

Reply:
I'll remove the privacy note and add something about p2sh-segwit.

Just bear in mind the current wallet doesn't tell the user what p2sh-segwit is either. The use of SegWit is essentially hidden from the user. It "just works".  It's bech32 that needs explaining, not the default behavior. I think the reason people may want to turn it on or off is because "they told me [not] to give them a bech32 address", rather than "they told me [not] to give them p2sh-segwit address".